video/video_decoder.cpp

```diff
@@ -631,10 +632,11 @@ Audio::Timestamp VideoDecoder::FixedRateVideoTrack::getDuration() const {
 }
 
 VideoDecoder::AudioTrack::AudioTrack(Audio::Mixer::SoundType soundType) :
-		_volume(Audio::Mixer::kMaxChannelVolume),
```

Comment:
Please revert this unnecessary indentation change

Reply:
done.